libraries/AP_HAL_ChibiOS/hwdef/common/stm32_util.h

```diff
@@ -59,6 +59,10 @@ void memory_flush_all(void);
 void stm32_set_utc_usec(uint64_t time_utc_usec);
 uint64_t stm32_get_utc_usec(void);
 
+void sntp_set_system_time_us(uint32_t sec, uint32_t us);
+void sntp_get_system_time_us(uint32_t* sec_, uint32_t* usec_);
```

Comment:
Why not return a struct of some time type? It's more efficient than two pointers and allows `const`

Reply:
See 35dda812ae4b45080fc05742c7da086aeb8f4409 as a recently merged  example of changing this.

Reply:
i thing we should drop sntp for now